bundles/framework/layerselector2/Flyout.js

```diff
@@ -289,12 +292,19 @@ Oskari.clazz.define('Oskari.mapframework.bundle.layerselector2.Flyout',
             me.tabContainer = Oskari.clazz.create(
                 'Oskari.userinterface.component.TabContainer'
             );
+
+            // Add filter tab change listener
+            me.tabContainer.addTabChangeListener(function(previousTab, newTab) {
```

Comment:
Adds a new listener with each filter request? Looks like a memory leak. Can you check this?

Reply:
Nevermind - this look ok. GitHub diff UI looked like this was done in enableFilter().